.gitignore

```diff
@@ -5,7 +5,6 @@ __pycache__/
 icecube/dataset/*
 !icecube/dataset/.gitkeep
 icecube/datacubes/*
-tests/resources/labels/dummy_mask_labels.json
```

Comment:
This will lead to creating redundant labels on remote with changing local paths as developers work on it